.eslintrc

```diff
@@ -4,6 +4,7 @@
     "sourceType": "script"
   },
   "rules": {
-    "valid-jsdoc": 0
+    "valid-jsdoc": 0,
+    "no-console": 0
```

Comment:
I don't think you need this, in any case it's better to disable the rule for a particular line of code — it's pretty useful to catch debug code.